packages/amplify_datastore/lib/amplify_datastore.dart

```diff
@@ -45,6 +52,7 @@ class AmplifyDataStore extends DataStorePluginInterface {
     if (provider == null || provider.modelSchemas.isEmpty) {
       throw ArgumentError("Need to provide at least one modelSchema");
     }
+    streamWrapper.registerModelsForHub(provider);
     return _instance.configureModelProvider(modelProvider: modelProvider);
```

Comment:
not you change, but it seems that we should use `provider` instead of `modelProvider` in the `configureModelProvider` call? cc: @fjnoyp